x/staking/keeper/unbonding.go

```diff
@@ -44,30 +44,14 @@ func (k Keeper) DeleteUnbondingIndex(ctx context.Context, id uint64) error {
 // GetUnbondingType returns the enum type of unbonding which is any of
 // {UnbondingDelegation | Redelegation | ValidatorUnbonding}
 func (k Keeper) GetUnbondingType(ctx context.Context, id uint64) (unbondingType types.UnbondingType, err error) {
```

Comment:
Consumers of this API will get a new error, this should be precise in the changelog or keep returning `ErrNoUnbondingType` when the error is `collections.ErrNotFound` for a nicer less API breakage